core/client/src/test/java/alluxio/client/file/options/MountOptionsTest.java

```diff
@@ -40,6 +41,10 @@ public void fieldsTest() {
     Assert.assertTrue(options.isReadOnly());
     options.setReadOnly(false);
     Assert.assertFalse(options.isReadOnly());
+    options.setShared(true);
+    Assert.assertTrue(options.isShared());
+    options.setShared(false);
+    Assert.assertFalse(options.isShared());
```

Comment:
Doesn't need to be in this PR, but I think this test should be split up into multiple tests? @aaudiber